flow/envs/green_wave_env.py

```diff
@@ -335,6 +335,37 @@ def _split_edge(self, edge):
         else:
             return 0
 
+    def _get_relative_node(self, agent_id, direction):
+        """Yield node number of traffic light agent in a given direction.
```

Comment:
Can we add some descriptiveness to this docstring? I'm actually not totally sure what the "node number in a given direction" is and it's making it sort of hard to edit this one.